runtime/v2/shim/shim_windows.go

```diff
@@ -113,12 +113,11 @@ func serveListener(path string) (net.Listener, error) {
 
 func handleSignals(logger *logrus.Entry, signals chan os.Signal) error {
 	logger.Info("starting signal loop")
+
 	for {
-		select {
-		case s := <-signals:
+		for s := range signals {
 			switch s {
 			case os.Interrupt:
-				break
```

Comment:
Should this break still be there? Looks like it's now only looping, but never breaking the loop (perhaps I don't understand the code 😊 )

Reply:
@fuweid commented on this in the original PR; The Golang answer is: *A "break" statement terminates execution of the innermost "for", "switch" or "select" statement.*

So in this case the `break` has no impact as it is breaking the `switch s` and the `for s := range signals {` was going to continue anyway with or without the `break`; at least that was my reading.

If the Windows folks want to break out of the switch and/or select, then a `break [Label]` is probably required? @jterry75

Reply:
I think this was probably supposed to be a return, but we should PR that fix separately. This change looks to be an accurate refactoring of the previous code.